app/src/marketplace/components/ProductPage/CollapsedText/collapsedText.stories.jsx

```diff
@@ -35,8 +35,7 @@ const longText = [
 ].join('\n\n')
 
 // from https://raw.githubusercontent.com/adamschwartz/github-markdown-kitchen-sink/master/TEST.md
-// $FlowFixMe
-const markdown = String.raw`This is a paragraph.
+const markdown = `This is a paragraph.
```

Comment:
do you know why this was using `String.raw` before?

Reply:
@juhah, can you, um, tag in? I didn't spot any difference between this and the previous one.

Reply:
Here it shouldn't make a difference, basically `raw` let's escape characters go through. Eg. `\n` is preserved but without raw would be converted to newline.

Reply:
I'll leave it like this then.